crates/sargon/src/profile/v100/entity/account/account.rs

```diff
@@ -136,6 +136,7 @@ impl Ord for Account {
                 .derivation_path()
                 .last_component()
                 .cmp(r.transaction_signing.derivation_path().last_component()),
+            _ => Ordering::Equal, // TODO check that
```

Comment:
TODO:
What the order should be when one of the accounta is securified?

Reply:
we should probably remove `Ord` for `Account` and use whatever order they were saved, which JSON should retain...